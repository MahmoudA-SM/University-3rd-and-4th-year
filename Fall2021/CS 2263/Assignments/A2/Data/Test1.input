u 15
u 20
o 
o
o
l
k
u 28
e
x
