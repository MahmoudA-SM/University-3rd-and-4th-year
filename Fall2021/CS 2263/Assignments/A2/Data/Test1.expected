15
20
20
15
failed pop
invalid instruction l
invalid instruction k
28
28
Successfully executed 6 instructions
|_ 28 _|
1 elements
